src/form/ColormapAllSteps.tsx

```diff
@@ -115,9 +115,36 @@ export const ColormapAllSteps: React.FC<Props> = (props) => {
               <div
                 style={{position: "relative"}}
               >
-                <button 
-                  onClick={()=> {if (visiblepickerIndex === ind) {setVisiblepickerIndex(null)} else {setVisiblepickerIndex(ind)} }}
-                  type="button"
+                {/* 
+                  following div should act as a button.
+                  Button was turned into div, because it needs nested divs for the colorpicker.
+                  These divs need to be nested because clicking on them need to keep this div in focus 
+                  The div now namely uses a onblur to close the colorpicker (as replacement of the overlay with onclick previously used)
```

Comment:
In my opinion, I think that you don't need to keep this div on focus all the time while clicking on the color map picker and it would look nicer if you don't show the outline at all (so `outline: none` for the button).  In this case, you can still use the button element instead of this div. Anyway, it's just a matter of preference, your solution is also working well.

Reply:
The focus is needed for the reason that a onblur closes tho colorpicker.
So clicking outside the colorpicker closes it.
Previously clicking outside the colorpicker also closed it, but this was done with an extra div overlay, blocking all other content. That old solution became difficult to accomplish, because i had to change some of the layout (the layout bug that this ticket fixes). Making the overlay div work correctly with the scroll is harder then using div that is in focus so I chose the latter.